### PR TITLE
Update footer docs

### PR DESCRIPTION
**Pull request by Blackbaud-TimPepper**

### CODE REVIEW

File: src/app/content/footer/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h3>
   <ul>
     <li>
       <p>
-        <stache-code>copyrightLabel</stache-code> — This flag allows you to include additional copyright information after the current date. The default text is <stache-code>Blackbaud, Inc. All rights reserved.</stache-code>
+        <stache-code>copyrightLabel</stache-code> — This flag allows you to include additional copyright information after the current date. The default text is <stache-code>&copy; Blackbaud, Inc. All rights reserved.</stache-code>
```
Comment (Blackbaud-BrandonJones) [inline src/app/content/footer/index.html]: The &copy; is always in the html and not part of the updatable text.  The default text here should not include that symbol as it's not in the translation.

Comment (Blackbaud-BrandonJones) [reply src/app/content/footer/index.html]: https://github.com/blackbaud/stache2/blob/master/src/app/public/src/modules/footer/footer.component.html#L14

Comment (Blackbaud-TimPepper) [reply src/app/content/footer/index.html]: @Blackbaud-LindseyRix thoughts?

Comment (blackbaud-johnly) [reply src/app/content/footer/index.html]: Does that mean the © appears no matter what text is supplied? If so, I agree with Jones that we shouldn't include it in the default text. But I'd call it out in the preceding sentence: 

<stache-code>copyrightLabel</stache-code> — Specifies additional copyright information to display after the current date. A © symbol appears before the copyright information, and the default text is <stache-code>Blackbaud, Inc. All rights reserved.</stache-code>

Comment (Blackbaud-BrandonJones) [reply src/app/content/footer/index.html]: Correct, as this is copyRight text, the `c` shows up with whatever text you put in there.

Comment (blackbaud-johnly) [reply src/app/content/footer/index.html]: Coolio. If that's the case, do my tweaks to the property description work for everyone? (Just noticed that the stache-code formatting was stripped out of the comment, but I did intend for the code snippets to remain formatted.)

Comment (Blackbaud-BrandonJones) [reply src/app/content/footer/index.html]: I like that

Comment (Blackbaud-LindseyRix) [reply src/app/content/footer/index.html]: Yes, but how are we letting them know what will be there by default. What we had didn't actually tell the user that the copyright symbol was already included.

Comment (blackbaud-johnly) [reply src/app/content/footer/index.html]: Sorry, I'm not sure I follow. My suggested rewrite says "A © symbol appears before the copyright information, and..." Is that not enough to make it clear that the symbol appears by default?

Comment (Blackbaud-BrandonJones) [reply src/app/content/footer/index.html]: @Blackbaud-LindseyRix ☝️

Comment (Blackbaud-LindseyRix) [reply src/app/content/footer/index.html]: <stache-code>copyrightLabel</stache-code> — This flag allows you to include additional copyright information after the `&copy;  (year)`. The default text after `&copy; (year)`  is <stache-code>Blackbaud, Inc. All rights reserved</stache-code>.

Comment (Blackbaud-LindseyRix) [reply src/app/content/footer/index.html]: Something like that. It's not clear in the original version what is default text and what is required text.

Comment (Blackbaud-BrandonJones) [reply src/app/content/footer/index.html]: Works for me,

Comment (blackbaud-johnly) [reply src/app/content/footer/index.html]: Looked at this a little more, and it's really weird trying to make the language clear. I mean, "additional copyright information" isn't really correct. This property seems to be more about replacing default text than supplementing it. Also, if we're trying to make the property descriptions clear to a less technical audience, I think `&copy; (year)` is unclear. Anyway, I played around with the text some more, and this is what I'd suggest:

`copyrightLabel` — Specifies the text to display in the footer's copyright information. By default, the footer displays a © symbol and the current year followed by "Blackbaud, Inc. All rights reserved." This property allows you to replace the default text after the year with your own copyright text.

Comment (Blackbaud-LindseyRix) [reply src/app/content/footer/index.html]: I think it's clearer than what we had before, so let's try this version.

**Comment by blackbaud-johnly** [comment]
Hey @Blackbaud-LindseyRix, I updated the property description based on our back-and-forth on that property.

Also, not to belabor this PR, but the more I look at this topic, the more confusing I find it. For example, I just realized that based on the code sample, this is a configuration property, not a component. But we don't explain that in the topic overview. This seems like it's worth calling out explicitly. At bare minimum, it seems like the paragraph after the properties section should be up in the overview. Here's what I'd suggest for that paragraph: 


FWIW, I think the `nav` property could also use some love. Currently, its description is "This flag contains an items array, allowing you to include specific links in the footer.." ... That doesn't really fully explain the property, although the sample code should be sufficient guidance. I'd remove the extra period, use the verb-first sentence fragment style, and flesh this out a little to try to make it more understandable at a glance to a less technical audience. What do you think of changing this property to:

`nav` — Specifies the links to include in the footer. You specify each link within an an `items` array by using a `title` property to specify the link text and a `route` property to specify the link path.

Also FWIW, the `siteName` property description seems weird in that we don't say what it does and we say that it is a value whether you can specify the value in the why are we just . If this is just a value that is

**Comment by Blackbaud-LindseyRix** [comment]
For what it's worth, I actually feel like that about different places in our docs. I really think we could look more closely at the components page and make sure we have good samples with good explanation. I particularly find the configuration and data page to be needing the most improvement.

**Pull request merged**